src/main/java/org/cactoos/iterable/IterableOf.java

```diff
@@ -45,14 +45,16 @@
  * @param <X> Type of item
  * @since 0.12
  * @checkstyle ClassDataAbstractionCouplingCheck (550 lines)
+ * @todo #1169:30m This class uses Mapped in order to coerce wild-card type.
+ *  Research a more eloquent solution and remove usage of Mapped.
```

Comment:
@andreoss I see you replaced the usage of Mapped with a cast, so this todo should be updated I think. Actually, I'm wondering if casting is not simply acceptable here… WDYT?

Reply:
@victornoel Removed. It seems there is no a better solution than casting for that (which could be put a decorator)